render.go

```diff
@@ -0,0 +1,87 @@
+package render
+
+import (
+	"image/color"
+	"image/draw"
+	"math"
+
+	"github.com/benoitkugler/textlayout/fonts"
+	"github.com/go-text/typesetting/shaping"
+	"github.com/srwiley/rasterx"
+	"golang.org/x/image/math/fixed"
+)
+
+type Renderer struct {
+	FontSize, PixScale float32
```

Comment:
Can you document the expected units for these fields?

Reply:
Ah yes I forgot the docs sorry! Fixed now